MonMaraicher/Features/Map/MapViewModel.swift

```diff
@@ -17,25 +17,17 @@ final class MapViewModel: ObservableObject {
     @Published var userPosition: MapCameraPosition = .userLocation(fallback: .automatic)
 
     init() {
-        readJsonFarmers()
+        getFarmers()
     }
 
     func onViewAppear() {
         requestUserAuthorization()
     }
 
-    private func readJsonFarmers() {
-        let decoder = JSONDecoder()
-        decoder.keyDecodingStrategy = .convertFromSnakeCase
-        do {
-            if let bundlePath = Bundle.main.path(forResource: "Farmers", ofType: "json"),
-               let farmerJson = try String(contentsOfFile: bundlePath).data(using: .utf8) {
-                let farmers = try decoder.decode([Farmer].self, from: farmerJson)
-                self.allFarmers = farmers
-            }
-        } catch {
-            print("Error decoding: \(error)")
-        }
+    private func getFarmers() {
```

Comment:
The naming here is not good because when you write `get<Something>` we are expecting a return value. But your func is not returning anything... ex: `loadFarmers` is better